tox.ini

```diff
@@ -23,7 +23,7 @@ setenv =
 commands =
     bash -c ' \
         pip-compile \
-            --emit-index-url --emit-trusted-host --annotate --generate-hashes \
+            --emit-index-url --emit-trusted-host --annotate \
```

Comment:
Why ? It does not work when using `git` with a hash ?

Reply:
Yes